core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/DataFrame.kt

```diff
@@ -33,6 +35,8 @@ public interface DataFrame<out T> : Aggregatable<T>, ColumnsContainer<T> {
         public fun empty(nrow: Int = 0): AnyFrame = if (nrow == 0) Empty else DataFrameImpl<Unit>(emptyList(), nrow)
 
         public inline fun <reified T> emptyOf(): DataFrame<T> = createEmptyDataFrameOf(T::class).cast()
+
+        public fun empty(schema: DataFrameSchema): AnyFrame = schema.createEmptyDataFrame()
```

Comment:
Could you please provide minor KDocs for new method and group of similar methods (like empty)?

Reply:
Sure, provided one for `emptyOf` and `empty`